xbmc/storage/MediaManager.cpp

```diff
@@ -760,3 +764,38 @@ void CMediaManager::RemoveDiscInfo(const std::string& devicePath)
   if (it != m_mapDiscInfo.end())
     m_mapDiscInfo.erase(it);
 }
+
+bool CMediaManager::playStubFile(const CFileItem& item)
+{
+  // Figure out Lines 1 and 2 of the dialog
+  std::string strLine1, strLine2;
+  CXBMCTinyXML discStubXML;
+  if (discStubXML.LoadFile(item.GetPath()))
+  {
+    TiXmlElement* pRootElement = discStubXML.RootElement();
+    if (!pRootElement || StringUtils::CompareNoCase(pRootElement->Value(), "discstub") != 0)
+      CLog::Log(LOGERROR, "Error loading %s, no <discstub> node", item.GetPath().c_str());
+    else
+    {
+      XMLUtils::GetString(pRootElement, "title", strLine1);
+      XMLUtils::GetString(pRootElement, "message", strLine2);
+    }
+  }
+
+  // Use the label for Line 1 if not defined
+  if (strLine1.empty())
+    strLine1 = item.GetLabel();
+
+  if (HasOpticalDrive())
+  {
+#ifdef HAS_DVD_DRIVE
```

Comment:
Im guessing this ifdef isnt needed either?

Has optical drive does that check already.

https://github.com/xbmc/xbmc/blob/65c06a2d361b7dab414ae1af39204ccd20e84d19/xbmc/storage/MediaManager.cpp#L391-L398

Reply:
iirc dropping this ifdef failed android-builds

Reply:
Yeah when I didn't have the ifdef the android builds failed so that's why I added it